frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-condition-table/enrollment-condition-table.component.ts

```diff
@@ -31,7 +33,7 @@ export class EnrollmentConditionTableComponent implements OnChanges, OnInit {
   }
 
   ngOnInit() {
-    this.experimentService.experimentStatById$(this.experiment.id).subscribe(stat => {
+    this.experimentConditionSub = this.experimentService.experimentStatById$(this.experiment.id).subscribe(stat => {
```

Comment:
@YashilDepani Please update the variable name `this.experimentConditionSub` to `this.experimentStateSub`.

Reply:
ok, code updated.